### ISSUE TITLE
[7.4.0] RuntimeError: [Failure instance: Traceback: <class 'UnicodeDecodeError'>: 'utf-8' codec can't decode byte 0x91 in position 0: invalid start byte

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#5091]
From the error reporter (there are a few more `UnicodeDecodeError` reports with similar stack traces):

```
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 114, in on_read_data
RuntimeError: [Failure instance: Traceback: <class 'UnicodeDecodeError'>: 'utf-8' codec can't decode byte 0x91 in position 0: invalid start byte
Tribler\Core\Libtorrent\LibtorrentMgr.py:492:<lambda>
Tribler\Core\Libtorrent\LibtorrentMgr.py:543:check_metainfo
lib\site-packages\twisted\internet\defer.py:460:callback
lib\site-packages\twisted\internet\defer.py:568:_startRunCallbacks
--- <exception caught here> ---
lib\site-packages\twisted\internet\defer.py:654:_runCallbacks
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:92:on_got_metainfo
Tribler\Core\Utilities\unicode.py:48:recursive_unicode
Tribler\Core\Utilities\unicode.py:48:<dictcomp>
Tribler\Core\Utilities\unicode.py:48:recursive_unicode
Tribler\Core\Utilities\unicode.py:48:<dictcomp>
Tribler\Core\Utilities\unicode.py:52:recursive_unicode
]
```

**Comment by devos50** [comment]
Re-opening this issue due to the following new report (7.4.1):

```
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 114, in on_read_data
RuntimeError: [Failure instance: Traceback: <class 'UnicodeDecodeError'>: 'utf-8' codec can't decode byte 0xc3 in position 0: invalid continuation byte
Tribler\Core\Libtorrent\LibtorrentMgr.py:576:_task_process_alerts
Tribler\Core\Libtorrent\LibtorrentMgr.py:543:check_metainfo
lib\site-packages\twisted\internet\defer.py:460:callback
lib\site-packages\twisted\internet\defer.py:568:_startRunCallbacks
--- <exception caught here> ---
lib\site-packages\twisted\internet\defer.py:654:_runCallbacks
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:80:on_got_metainfo
Tribler\Core\Modules\MetadataStore\OrmBindings\torrent_metadata.py:36:tdef_to_metadata_dict
Tribler\Core\Category\Category.py:51:calculateCategory
]
```

**Comment by ichorid** [comment]
This is a different one. This one is about `calclulateCategory` method, while the first one was about `recursive_unicode`

**Comment by devos50** [comment]
```
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 114, in on_read_data
RuntimeError: [Failure instance: Traceback: <class 'UnicodeDecodeError'>: 'utf-8' codec can't decode byte 0xe0 in position 3: invalid continuation byte
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:149:render_GET
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:135:on_magnet
lib\site-packages\twisted\internet\defer.py:322:addCallback
lib\site-packages\twisted\internet\defer.py:311:addCallbacks
--- <exception caught here> ---
lib\site-packages\twisted\internet\defer.py:654:_runCallbacks
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:92:on_got_metainfo
Tribler\Core\Utilities\unicode.py:48:recursive_unicode
Tribler\Core\Utilities\unicode.py:48:<dictcomp>
Tribler\Core\Utilities\unicode.py:48:recursive_unicode
Tribler\Core\Utilities\unicode.py:48:<dictcomp>
Tribler\Core\Utilities\unicode.py:50:recursive_unicode
Tribler\Core\Utilities\unicode.py:50:<listcomp>
Tribler\Core\Utilities\unicode.py:48:recursive_unicode
Tribler\Core\Utilities\unicode.py:48:<dictcomp>
Tribler\Core\Utilities\unicode.py:53:recursive_unicode
]
```

**Comment by devos50** [comment]
You should pass the `ignore_errors` parameters in the recursive call to `recursive_unicode`.

**Comment by ichorid** [comment]
solved by #5120
